test/Microsoft.ApplicationInsights.AspNetCore.Tests/TelemetryInitializers/WebSessionTelemetryInitializerTests.cs

```diff
@@ -1,20 +1,20 @@
-﻿namespace Microsoft.ApplicationInsights.AspNet.Tests.TelemetryInitializers
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Tests.TelemetryInitializers
 {
     using System;
-
-    using Microsoft.ApplicationInsights.AspNet.TelemetryInitializers;
-    using Microsoft.ApplicationInsights.AspNet.Tests.Helpers;
+    using System.Threading.Tasks;
```

Comment:
In another AI project I noticed #ifdefs around this using, are we going to run into that with the framework versions supported in this library?